crates/shared/src/http_solver/model.rs

```diff
@@ -325,12 +340,18 @@ mod execution_plan_internal {
     }
 }
 
-#[derive(Clone, Debug, Deserialize, PartialEq, Eq, PartialOrd, Ord, Serialize)]
+#[derive(Clone, Debug, Default, Deserialize, PartialEq, Eq, PartialOrd, Ord, Serialize)]
 pub struct ExecutionPlanCoordinatesModel {
     pub sequence: u32,
     pub position: u32,
 }
 
+#[derive(Clone, Debug, Default, Deserialize, PartialEq, Eq, Serialize)]
+pub struct CoordinatesWithInternal {
+    pub coordinates: ExecutionPlanCoordinatesModel,
+    pub internal: bool,
```

Comment:
Actually, instead can't we just make `ExecutionPlanCoordinatesModel` have an `internal` flag with a `#[serde(default)]` so it will accept when its missing?

Reply:
Sure, I am ok with flattening everything to single struct. Done.

Reply:
When we merge https://github.com/cowprotocol/services/pull/791 I will just rename `ExecutionPlanCoordinatesModel ` to `ExecutionPlan` and remove the `#[serde(default)]` for `internal` field.